it/test/uk/gov/hmrc/agentclientrelationshipsfrontend/controllers/AgentFastTrackControllerSpec.scala

```diff
@@ -265,7 +265,7 @@ class AgentFastTrackControllerSpec extends ComponentSpecHelper with AuthStubs wi
       val result = post(fastTrackRoutes.AgentFastTrackController.agentFastTrack.url)(toFastTrackRequests(fastTrackFormData))
       result.status shouldBe SEE_OTHER
 
-      result.header("Location").value shouldBe journeyRoutes.JourneyErrorController.show(journeyType, "known-fact-not-matched").url
+      result.header("Location").value shouldBe journeyRoutes.JourneyErrorController.show(journeyType, "client-not-found").url
```

Comment:
this second parameter to the route has recently changed and should be a JourneyErrorType instead of a string - so the value here would be `JourneyErrorType.NotFound` instead of "client-not-found"